quipucords/api/reports/view.py

```diff
@@ -34,35 +31,33 @@
 @authentication_classes(auth_classes)
 @permission_classes(perm_classes)
 @renderer_classes((ReportsGzipRenderer,))
-def reports(request, report_id=None):
+def reports(request, report_id):
     """Lookup and return reports."""
     reports_dict = {}
-    if report_id is not None:
-        if not is_int(report_id):
-            error = {"report_id": [_(messages.COMMON_ID_INV)]}
-            raise ValidationError(error)
```

Comment:
this part of the code is unreachable given how the URL for this view is set.

let Django do it's job and move on